plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -505,6 +556,17 @@ public static LocalDateTime fromPrestoTimestamp(long epochMicros)
         return LocalDateTime.ofInstant(instant, UTC);
     }
 
+    public static LocalDateTime fromLongTimestamp(LongTimestamp timestamp, int precision)
+    {
+        long epochSeconds = floorDiv(timestamp.getEpochMicros(), MICROSECONDS_PER_SECOND);
+        int microsOfSecond = floorMod(timestamp.getEpochMicros(), MICROSECONDS_PER_SECOND);
+        long picosOfMicro = round(timestamp.getPicosOfMicro(), MAX_PRECISION - precision);
+        int nanosOfSecond = (microsOfSecond * NANOSECONDS_PER_MICROSECOND) + (int) (picosOfMicro / PICOSECONDS_PER_NANOSECOND);
```

Comment:
`(int)` -> toIntExact 

to avoid IDE warning

Reply:
Done!